script/index.js

```diff
@@ -22,7 +22,7 @@ const P1NAME = 'Sam';
 const P1CHARA = 'saml';
 
 let playerTurn = false; //declares a value used for tracking player turn.
-let logging //declares a value used to toggle logging
+let logging = true; //declares a value used to toggle logging
```

Comment:
"let logging = true; //declares a value used to toggle logging"

it should be 

let logging //declares a value used to toggle logging